scripts/perftune.py

```diff
@@ -738,21 +827,12 @@ def __max_rx_queue_count(self, iface):
         ixgbevf: VF NICs support up to 4 RSS queues.
         i40e:    PF NICs support up to 64 RSS queues.
         i40evf:  VF NICs support up to 16 RSS queues.
+        mlx:     PF/VF NICs throttle to respective cpu count dedicated to sq split due to si problems when cpu is shared with scylla, this is minimum
```

Comment:
Drop all changes in the `__max_rx_queue_count()` please. We don't know of any limitation in Mellanox NICs.

Reply:
could this be a limit?
https://github.com/torvalds/linux/blob/master/drivers/net/ethernet/mellanox/mlx4/mlx4_en.h#L74

not sure where to look in https://github.com/torvalds/linux/tree/master/drivers/net/ethernet/mellanox/mlx5/core

anyways, dropping, we can always add a ceiling, the important is to lower it now to the dedicated cpus

Reply:
Let me know when I can look at those changes.